filters/united-states/california/san-francisco/public-art-json-to-json.js

```diff
@@ -1,29 +1,82 @@
-const filter = function (data, std_lib, stringify, skip_errors) {
+const filter = function (data, std_lib, schema, validator, stringify) {
+    // check for standard library and pull out required functions
+    if (!std_lib) {
+        throw "standard library not provided";
+    }
+    let add_required = std_lib.get("add_required");
+    let add_if_not_null = std_lib.get("add_if_not_null")
+    let remove_if_empty = std_lib.get("remove_if_empty");
+    let validate_params = std_lib.get("validate_params");
+    
+    // validate parameters object
+    schema = validate_params(schema, validator);
+
     if (typeof data === 'string' || data instanceof String) {
         data = JSON.parse(data);
     }
     
     let new_data = [];
-
+    let errors = [];
+    
     data.map(d => {
         let item = {};
-        item.name = d.title;
-        if (item.name === undefined) {
-            console.log(`Data name not found for art with url ${d.url}`);
+        let skip = false;
+
+        let name;
+        let artist;
+
+        const title_field = d.title;
+        if (title_field != null && title_field.includes("by")) {
```

Comment:
may also want to check if its an empty string or undefined here

Reply:
There were no cases like that in their dataset, but good point